backend/curriculum_tracking/models.py

```diff
@@ -1383,3 +1385,31 @@ def add_collaborator(self, user, add_as_project_reviewer):
                 if self.recruit_project:
                     self.recruit_project.reviewer_users.add(user)
         self.save()
+
+    def finding_latest_reviewer_users(self):
+        """Everytime a person reviews the status will be changed, so look for any status
+        changes after a review-request and give the ids of those reviewers"""
+
+        reviewer_user_since_latest_review_request = []
+        review_requests_made = []
+
+        # Making sure to only get reviewers since the latest request for a review
+        if self.status in [
+            AgileCard.REVIEW_FEEDBACK,
+            AgileCard.IN_REVIEW,
+        ]:
+            review_requests_made.append(self.topic_progress.review_request_time)
+        else:
+            review_requests_made = []
+
+        reviews_since_latest_review_request = RecruitProjectReview.objects.filter(
+            recruit_project=AgileCard.recruit_project
+        ).filter(timestamp__lt=self.topic_progress.review_request_time)
```

Comment:
we need reviews that happened AFTER the last review request time. This fetches the reviews that happened before.